DESCRIPTION

```diff
@@ -19,7 +19,8 @@ Suggests:
   sandwich (>= 2.3-1),
   survival,
   testthat (>= 2.0.0),
-  Zelig (>= 5.0-16)
+  Zelig (>= 5.0-16),
+  biglm
```

Comment:
Please adhere to the alphabetical order here. Also: is it necessary to add a dependency here just for testing purposes?

Reply:
I wanted to put a test for the CI case, if you think it is not necessary I'll take it out

Reply:
Is it possible to test this with `ci.force = TRUE`?

Reply:
Would it work to keep the test but add a line that skips the test if the package is not installed? Something like `skip_if_not_installed("biglm", minimum_version = "1.0")` (+ adjustment of the version number)? I'm not sure if this is possible with the `biglm` package under `ENHANCES` instead of `SUGGESTS`, but you could try that out...

Reply:
no since `plotreg` does not take `ci.force = TRUE `, that was my first idea too.

Reply:
Just noticed that `biglm` wasn't even listed under `ENHANCES`. I guess you could try that first and add the skip line. If that's not possible, try the `ci.force` idea?

Reply:
sorry, what do you mean by "add the skip line"?

Reply:
Put `skip_if_not_installed("biglm", minimum_version = "1.0")` or something similar inside the `test_that` command but before the `expect` commands start.

Reply:
There are examples in the other test scripts.

Reply:
ok! thanks a lot!

Reply:
More specifically, there is a very similar example in the `test-extract.R` file in the `roxygenize-extract` branch. See [here](https://github.com/leifeld/texreg/blob/cb18163b73ac093a139fdd15af20704688470155/tests/testthat/test-extract.R#L5). Also add the `require` statement in the next line and add the package to the `ENHANCES` field in the `DESCRIPTION` file.